src/components/Categories/CategoryList/CategoryList.module.css

```diff
@@ -0,0 +1,74 @@
+.categoriesList {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.categoryItem {
+  height: 250px;
+  border-radius: 20px;
+  position: relative;
+  background: rgba(5, 5, 5, 0.18);
+  background-image: url('../../../temp/cat-img.jpg');
```

Comment:
should not work like this, use multiple backgrounds instead

Reply:
Corrected